test/steps/tag_steps.go

```diff
@@ -0,0 +1,33 @@
+package steps
+
+import (
+	"fmt"
+
+	"github.com/cucumber/godog"
+	"github.com/cucumber/messages-go/v10"
+)
+
+// TagSteps defines Gherkin step implementations around merges.
+func TagSteps(suite *godog.Suite, fs *FeatureState) {
+	suite.Step(`^I have the following tags$`, func(table *messages.PickleStepArgument_PickleTable) error {
```

Comment:
People don't have tags 🙂  Thoughts about renaming this to `my repo has the following tags`?